### ISSUE TITLE
Site Separtor

### ISSUE THREAD

**Issue by xevaballx** [xevaballx/wildlife#9]
We need to add a script that separates the sites in the training and validation sets.

from the competition "In order to make sure that our models predict well in new contexts, it is important that train and test sets have entirely different environments. In this case, we ensure this by making sure that **sites are entirely in the train set or the test set, but never in both**. We recommend that you take a similar approach in setting up your own splits of the training set."
